docs/guides/latest/workloads/deployment.md

```diff
@@ -0,0 +1,435 @@
+
+**Deploy Deployment:**
+
+Now, we have to deploy a new Deployment whether the backed up data has been restored.
+
+Below is the YAML of the Deployment that we are going to create,
+
+```console
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: demo-pvc
+  namespace: demo
+spec:
+  accessModes:
+  - ReadWriteOnce
+  storageClassName: standard
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: stash-demo
+  name: stash-recovered
+  namespace: demo
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: stash-demo
+  template:
+    metadata:
+      labels:
+        app: stash-demo
+      name: busybox
+    spec:
+      containers:
+      - args:
+        - sleep
+        - "3600"
+        image: busybox
+        imagePullPolicy: IfNotPresent
+        name: busybox
+        volumeMounts:
+        - mountPath: /source/data
+          name: source-data
+      restartPolicy: Always
+      volumes:
+      - name: source-data
+        persistentVolumeClaim:
+          claimName: demo-pvc
+```
+
+Let's create the deployment we have shown above.
+
+```console
+$ kubectl apply -f ./recovered_deployment.yaml 
+persistentvolumeclaim/demo-pvc created
+deployment.apps/stash-recovered created
+```
+
+Now, wait for deployment’s pod to go into the `Running` state.
+
+```console
+$ kubectl get pod -n demo 
+NAME                               READY   STATUS    RESTARTS   AGE
+stash-recovered-676bd87957-7bcfg   1/1     Running   0          31s
+stash-recovered-676bd87957-pv4hg   1/1     Running   0          31s
+stash-recovered-676bd87957-q2hbl   1/1     Running   0          31s
+
+```
+
+**Create RestoreSession:**
+
+We need to create a `RestoreSession` crd to restore the backed up data in the target volume from backend.
+
+Below is the YAML of the `RestoreSesion` crd that we are going to create,
+
+```console
+apiVersion: stash.appscode.com/v1beta1
+kind: RestoreSession
+metadata:
+  name: deployment-restore
+  namespace: demo
+spec:
+  repository:
+    name: local-repo
+  rules:
+    - paths:
+        - /source/data
+  target: # target indicates where the recovered data will be stored
+    ref:
+      apiVersion: apps/v1
+      kind: Deployment
+      name: stash-recovered
+    volumeMounts:
+      - name:  source-data
+        mountPath:  /source/data
+```
+
+Here,
+
+`spec.repository.name` specifies the `Repository` crd that holds the backend information where our backed up data has been stored.
+
+`spec.target.ref` refers to the target workload where the recovered data will be stored.
+
+Let's create the `RestoreSession` crd we have shown above,
+
+```console
+$ kubectl apply -f ./restoresession.yaml
+restoresession.stash.appscode.com/deployment-restore created
+```
+
+Once, you have created the `RestoreSession` crd, Stash will create a job to restore. We can watch the `RestoreSession` phase to check if the restore process is `succeeded` or not.
```

Comment:
Stash will not create Job. It will inject an init-container to restore.

Reply:
Change this para to:

Once, you have created the `RestoreSession` crd, Stash will inject `init-container` to `stash-recovered` Deployment. The deployment will restart and the init-container will recovered on start-up.

**Verify Init-Container:**
<Use `kubectl describe deployment/stash-recovered -n demo` to show that the init-container has been injected.>

Reply:
**Wait for RestoreSession to Succeeded:**